build_nark/main.go

```diff
@@ -0,0 +1,235 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// This file has been modified by Enfabrica Corp. from the version obtained from
+// the above copyright holder.
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"html/template"
+	"mime/quotedprintable"
+	"os"
+	"strings"
+
+	"github.com/enfabrica/enkit/lib/logger"
```

Comment:
Common convention (and the way it is in internal) is

- stdlib
- third party 
- local

Doesn't look like enkit repo really has any consistency though

Reply:
Done